src/PrestaShopBundle/Resources/config/services/core/configuration.yml

```
@@ -9,6 +9,10 @@ services:
     class: 'PrestaShop\PrestaShop\Core\Configuration\PhpExtensionChecker'
     public: true
 
+  prestashop.core.configuration.avif_extension_checker:
+    class: 'PrestaShop\PrestaShop\Core\Configuration\AvifExtensionChecker'
+    public: true
```

```suggestion
  PrestaShop\PrestaShop\Core\Configuration\AvifExtensionChecker: ~
```
Suggestion but for this new service it's easy to rely on FQCN service definition from the start since it's our new/future way of defining services